tests/virt_autotest/vgpu.pm

```diff
@@ -132,10 +135,15 @@ sub install_vgpu_manager_and_reboot {
             return;
         }
     }
+
+    # Remove NVIDIA open driver
```

Comment:
How about linking the bug id here in comment so that we can recall where it comes from ?

Reply:
Sound good.